skbeam/core/correlation.py

```diff
@@ -769,3 +769,29 @@ def _validate_and_transform_inputs(num_bufs, num_levels, labels):
 
     return (label_array, pixel_list, num_rois, num_pixels,
             lag_steps, buf, img_per_level, track_level, cur)
+
+
+def one_time_from_two_time(two_time_corr):
+    """
+    This will provide the one-time correlation data from two-time correlation
+    data. 
+
+    Parameters
+    ----------
+    two_time_corr : array
+        matrix of two time correlation
+        shape (number of images, number of images, number of labels(ROI))
```

Comment:
After in-person discussion with @sameera2004 , we have concluded that the two-time correlation results, that is `g2`, should have a shape of `(num ROI, num images, num images)` which will require touching the `_init_state_two_time()` function, and the `two_time_state_to_results()` function and probably the tests and the examples.... (sorry :cry: )